app/views/posts/index.html.erb

```diff
@@ -1,2 +1,31 @@
-<h1>Posts#index</h1>
-<p>Here is a list of posts for a given user</p>
+<div class="flex flex-col w-full">
+<div class="flex items-center justify-between flex-col w-full p-6 bg-white rounded-lg md:flex-row">
+
+  <div class="w-20 mr-2">
+  <%= image_tag 'img2.jpg', alt: 'icon'%>
+  </div>
+
+
+  <div class="flex justify-between w-full">
+    <%# <h2 class="flex text-bold text-2xl"><%= @post.author %></h2> %>
+    <%# <div>Number of Posts: <%= @post.comments_counter%> </div> %>
+  </div>
+
+<ul class="w-full max-w-4xl mx-auto p-6 space-y-6">
+  <% @latest_posts.each do |post| %>
+  
+    <li class="flex items-center justify-between flex-col w-full p-6 bg-white rounded-lg md:flex-row">
+      <a href="http://127.0.0.1:3000/users/<%= post.author_id%>/posts/<%= post.id%>" class="flex w-full">
+        <div class="flex justify-between w-full"><h2 class="flex text-bold text-2xl"><%= post.title%></h2>
+        <ul class="flex justify-between w-48">
+          <li>Comments: <%= post.comments_counter%></li>
+          <li>Likes: <%= post.likes_counter%></li>
+        </ul>
+        </div>
+      </a>
+    </li>
+  <% end %>
+</ul>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use `latest_comments` method that you created in the `post` model.
[Hint] You suppose to use it like that:
```ruby
<% post.latest_comments.each do |comment| %>
```